task/06-conditions-n-loops-tasks.js

```diff
@@ -270,7 +321,10 @@ function isCreditCardNumber(ccn) {
  *   165536 (1+6+5+5+3+6 = 26,  2+6 = 8) => 8
  */
 function getDigitalRoot(num) {
-    throw new Error('Not implemented');
+    let numArr = ('' + num).split('').map(function(digit)  {return + digit;});
+    const reducer = (total, item) => total + item;
+    let sumArr = ('' + numArr.reduce(reducer)).split('').map(function(digit)  {return + digit;});
+    return sumArr.length > 1 ? sumArr[0] + sumArr[sumArr.length-1] : sumArr[0];
```

Comment:
avoid this code style, if-else is more readable than multiple conditional operators ? :